src/Tablebot/Plugins/Cats.hs

```diff
@@ -0,0 +1,68 @@
+-- |
+-- Module      : Tablebot.Plugins.Cats
+-- Description : A very simple plugin that provides cat pictures.
+-- Copyright   : (c) Finnbar Keating 2021
+-- License     : MIT
+-- Maintainer  : finnjkeating@gmail.com
+-- Stability   : experimental
+-- Portability : POSIX
+--
+-- This is an example plugin which just responds with a cat photo to a .cat call
+module Tablebot.Plugins.Cats (catPlugin) where
+
+import Control.Monad.IO.Class (MonadIO (liftIO))
+import Data.Aeson (FromJSON, eitherDecode)
+import Data.Functor ((<&>))
+import Data.Text (Text, pack)
+import GHC.Generics (Generic)
+import Network.HTTP.Conduit (Response (responseBody), parseRequest)
+import Network.HTTP.Simple (httpLBS)
+import Tablebot.Plugin.Discord (sendMessage)
+import Tablebot.Plugin.Parser (noArguments)
+import Tablebot.Plugin.Types (Command (Command), Plugin (commands), plug)
+
+-- | @CatAPI@ is the basic data type for the JSON object that thecatapi returns
+data CatAPI = CatAPI
+  { breeds :: ![Text],
+    id :: !Text,
+    url :: !Text,
+    width :: !Int,
+    height :: !Int
+  }
+  deriving (Show, Generic)
+
+instance FromJSON CatAPI
+
+-- | @cat@ is a command that takes no arguments (using 'noArguments') and
+-- replies with an image of a cat. Uses https://docs.thecatapi.com/ for cats.
```

Comment:
[docs.thecatapi.com](https://docs.thecatapi.com/) suggests that an API key is needed, except none is used in this code (but it still succeeds) - are we going to get rate-limited because we're not using a key?

Reply:
Huh. We might be. I'll have a look into it.

Reply:
Ok, I am making changes to solve this, but it will require adding another entry to the .env so my cat api key isn't leaked

Reply:
Added.